src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj

```diff
@@ -10,102 +17,60 @@
     <TargetFramework>net472</TargetFramework>
 
     <!-- VSIX -->
-    <GeneratePkgDefFile>true</GeneratePkgDefFile>
-    <IncludeAssemblyInVSIXContainer>true</IncludeAssemblyInVSIXContainer>
+    <GeneratePkgDefFile>false</GeneratePkgDefFile>
+    <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
+    <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
     <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
     <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
+    <ImportVSSDKTargets>true</ImportVSSDKTargets>
+    <IsProductComponent>true</IsProductComponent>
     <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot>
     <ExtensionInstallationFolder>Microsoft\ManagedLanguages\VBCSharp\InteractiveComponents</ExtensionInstallationFolder>
 
     <!-- VS Insertion -->
     <VisualStudioInsertionComponent>Microsoft.CodeAnalysis.LanguageServices</VisualStudioInsertionComponent>
   </PropertyGroup>
   <ItemGroup Label="Project References">
-    <ProjectReference Include="..\..\Interactive\Host\InteractiveHost32.csproj">
-      <Ngen>false</Ngen>
+    <ProjectReference Include="..\..\Interactive\DesktopHost\InteractiveHost32.csproj">
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup;BuiltProjectOutputGroupDependencies;GetCopyToOutputDirectoryItems</IncludeOutputGroupsInVSIX>
```

Comment:
Isn't this still going to include binaries in this VSIX? This seems to be backwards from what I would have assumed which this is now empty per the comment at the top?

Reply:
I guess we can remove the binaries.

Reply:
This is probably just a bad merge :(

Reply:
Ah, no I think we need to keep these here. That's the point of the project - it collects all dependencies to `VSIXSourceItem`. This list is then returned from `VsixSourceItemsOutputGroup` target to the project that includes them.

Reply:
Oh. Why aren't we just putting the project references for InteractiveHost that are here directly in the setup package then that is consuming the output group? The layer of indirection seems odd, especially because this isn't just pulling in direct InteractiveHost contents either -- since this is also referencing things like Microsoft.CodeAnalysis.CSharp.dll and *also* apparently including them, it means we're ending up with two paths where files end up in the final setup package.

Reply:
> it means we're ending up with two paths where files end up in the final setup package.

That is indeed the goal :)

Reply:
But then we can't delete this project? Then it seems sad.

Reply:
Right, the project is needed.